source/hwIo/__init__.py

```diff
@@ -0,0 +1,20 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2015-2021 NV Access Limited, Babbage B.V.
+
+
+"""Raw input/output for braille displays via serial and HID.
+See the L{Serial} and L{Hid} classes.
+Braille display drivers must be thread-safe to use this, as it utilises a background thread.
+See L{braille.BrailleDisplayDriver.isThreadSafe}.
+"""
+
+
+from .base import (  # noqa: F401

```

Comment:
Have you considered placing these in `__all__` rather than silencing linter warning?

Reply:
It means they would all have to be mentioned twice in the file. Once when importing, and once (as a string) in `__all__`. Makes for more upkeep I'd say, unless we wanted the specific bennifits of `__all__` other than just making the linter happy, which I'm not even sure it would? perhaps it might.